tests/graph_tests.py

```diff
@@ -0,0 +1,35 @@
+from dsalgo.graph_traversal import Graph
+
+mat = [
+    [1, 3],
+    [1, 5],
+    [1, 6],
+    [2, 5],
+    [2, 6],
+    [3, 4],
+    [3, 5],
+    [5, 6],
+    [7, 6],
+    [9, 8],
+    [8, 10],
+    [9, 4],
+    [4, 8],
+    [10, 2]]
+
+g = Graph()
+b = Graph()
+#Test for directed graph
+for i in mat:
+    g.addEdge(i[0], i[1])
+    b.addEdge(i[0], i[1])
+    b.addEdge(i[1], i[0])
```

Comment:
reason for this?

Reply:
G is directed graph and B is undirected graph that means if you can go from A to B then you can also go from B to A